### PR TITLE
feat(install): add fxa-email-server to servers

### LINKED ISSUE #111: Add fxa-email-service to servers
Ref: https://github.com/mozilla/fxa-email-service

### PR DESCRIPTION
**Pull request by brizental**
Fixes #111 

r? @vladikoff 

Question: Should we turn off rocket logs for `dev` too? They kinda get in the way, in my opinion.

### CODE REVIEW

**Comment by vladikoff** [comment]
> Question: Should we turn off rocket logs for dev too? They kinda get in the way, in my opinion.

No, we want as much logging as possible in dev

File: _scripts/check.sh [hunk 1]
```diff
@@ -1,4 +1,16 @@
 #!/usr/bin/env bash
+if [[ $(which rustc) ]]; then
+  if  [[ ! $(rustc --version) =~ "nightly" ]]; then
+    echo "make rust nightly your default to continue installation"
+    echo "https://github.com/rust-lang-nursery/rustup.rs#working-with-nightly-rust"
+    exit 1
+  fi
+else
+  echo "install rust to continue installation"
```
Comment (vladikoff) [inline _scripts/check.sh]: can we call `rustup` here? https://rustup.rs/

File: servers.json [hunk 2]
```diff
@@ -34,6 +34,16 @@
       "max_restarts": "1",
       "min_uptime": "2m"
     },
+    {
```
Comment (vladikoff) [inline servers.json]: please put this in the end of this array of servers

**Comment by vladikoff** [comment]
@brizental we need to add Rust in .travis.yml for CI to pass https://travis-ci.org/mozilla/fxa-local-dev/builds/403703069#L650

**Comment by vladikoff** [comment]
Got a 

```

For more information about this error, try `rustc --explain E0658`.
error: Could not compile `proc-macro2`.
warning: build failed, waiting for other jobs to finish...
error: build failed
``` 

on the latest build :(

**Comment by shane-tomlinson** [comment]
\o/

File: .travis.yml [hunk 3]
```diff
@@ -38,6 +38,15 @@ install:
   - npm install -g npm@6
   - node --version
   - npm --version
+  - curl -sSf https://sh.rustup.rs | sh -s -- -y --verbose
+  - PATH=$PATH:$HOME/.cargo/bin
+  - rustup install nightly-2018-06-23
```
Comment (philbooth) [inline .travis.yml]: Late to this, sorry, but wouldn't it make sense to use the same version we specify for CI and docker? (currently 2018-07-14)

Comment (vladikoff) [reply .travis.yml]: Yeap good idea!

**Pull request merged**